### PR TITLE
Ports Gimmick Traitor from Daedalus Dock (and raises starting TC)

### PR DESCRIPTION
**Pull request by nevimer**
## About The Pull Request
mstsc_mBeessMCFA
mstsc_821RI1VnPn

ports the meat from https://github.com/DaedalusDock/daedalusdock/pull/564

## Why It's Good For The Game
I don't think anyone but the player of traitors enjoys progressive traitor. This replaces it with the gimmick traitor goals, which are much more entertaining for everyone. Big QOL.

## Changelog

:cl:
add: Traitor Most traitor goals have been replaced with gimmick traitor goals. 
/:cl:

### CODE REVIEW

**Comment by Serxule** [comment]
does this completely remove progtot? im fairly certain this will be less fun for everyone by restricting traitors

**Comment by mcmeiler** [comment]
Some of these gimmicks literally break server rules

**Comment by Serxule** [comment]
and yeah, this is insanely fucking restrictive on traitors, makes it so they cant do anything more interesting than stealth greentexting or murderboning

**Comment by Serxule** [comment]
you also STILL NEED REPUTATION TO BUY A BUNCH OF SHIT, OF WHICH YOU CANT GET WITH NO OBJECTIVES

**Comment by Serxule** [comment]
legitmately, who gives any flying fucks about greentexting? if an antag only cares about greentexting that isnt fun, this literally just reverts antags back to nrp greentexters

**Comment by mcmeiler** [comment]
TL;DR
> can't get most items because of rep
> objectives are straight up banbait in some cases

Incredible work

**Comment by nevimer** [comment]
> TL;DR
> 
> > can't get most items because of rep
> > objectives are straight up banbait in some cases
> 
> Incredible work

I don't know why you believe you can't, you get all antag items unlocked in an hour or less passively with the rep system. This didn't remove the rep system, which is in place to prevent round start massacres. 
If you think the objectives are ban bait, then you need to rethink what they mean instead of going for the automatic worst assumption.

**Comment by StrangeWeirdKitten** [comment]
I think Progression Traitor is an awful game design choice because instead of roleplay, you're incentivized to basically snowball. You should have a fixed TC amount. Multiple other servers do this, and it's much more entertaining than a traitor blowing up the ORM round start and killing all the station pets. Our rounds are 2:30 hours long. Reputation itself is bound to how long the round has been going. People who point a gun at their boss are leagues more entertaining than people hacking a comm console for the 100th time inside a maintenance shaft for free 10 TC. The fact you're getting 25 TC is **generous** in itself.

image

You can buy this after 30 minutes. I don't see the issue here. Most of the other items need not as much reputation.

image

You can rewrite your objective to literally anything.

**Comment by Majkl-J** [comment]
This is basically OPFOR except quirkier and with no admin approval/limitations

**Comment by Tupinambis** [comment]
Reputation system doesn't make sense without secondary objectives. I don't want to wait 20-30 minutes to do anything.

**Comment by StrangeWeirdKitten** [comment]
> Reputation system doesn't make sense without secondary objectives. I don't want to wait 20-30 minutes to do anything.

You get 10 points per minute of reputation. You unlock the d-sword in thirty minutes. You don't need a d-sword before most everyone is set up, mostly including medical. Each round is 2:30 hours, typically.

**Comment by Tupinambis** [comment]
> > Reputation system doesn't make sense without secondary objectives. I don't want to wait 20-30 minutes to do anything.
> 
> You get 10 points per minute of reputation. You unlock the d-sword in thirty minutes. You don't need a d-sword before most everyone is set up, mostly including medical. Each round is 2:30 hours, typically.

Problem is it also basically means you have to wait, because your best self defense items are locked behind 20-30 minutes and why the heck would you start doing some stuff without them. That's not fun for antags or the station honestly. It's basically encouraging antags to just afk.

**Comment by destrucktoid** [comment]
Progressive traitor allows you to take risks to gain rewards and in doing so create conflict. Most antags don't really have a plan for a gimmick or something they specifically want to do in a shift, progressive traitor gives them something lowish level as a risk that can create conflict in exchange for a reward. Removing the entirety of progression traitor whilst keeping the rep requirements for some ungodly reason and only giving a 5 TC increase is not a good change. 
Progressive traitor's main benefit is the fact you can do things to earn more TC, ergo allowing you to get stronger as the shift goes on, which is what most solo antags are supposed to do. Simultaneously, the rep system is implemented so that traitors who want to get the big guns need to do objectives to get to them quicker, removing it and replacing it with nothing is completely stupid.

**Comment by YakumoChen** [comment]
> I don't think anyone but the player of traitors enjoys progressive traitor. 

This sounds most of all like you don't want traitors to have fun. 

The station should feel antagonised by sabotage and security enjoys having stuff to investigate, imo. This just makes it a slightly better OPFOR because we actually get antags

**Comment by StrangeWeirdKitten** [comment]
We've been running this system for the better part of a month now and I'm going to grab official feedback from the server staff.

**Comment by StrangeWeirdKitten** [comment]
rip prog-tot on /tg/
